pages/documentation/migration-V3.md

```diff
@@ -0,0 +1,822 @@
+
+You'll need to update any instances of the small search button on your site. We're now using explicit images to better support legibility in instances where icons do not load. 
+
+###### What to do
+1. Check your codebase for instances of `<span class="usa-sr-only">Search</span>`.
+1. Update the markup from the old version to the new version if you use it.
+
+###### Old
+
+{:.site-terminal}
+```html
+<button class="usa-button" type="submit">
+  <span class="usa-sr-only">Search</span>
+<button>
+```
+
+###### New
+
+{:.site-terminal}
+```html
+<button class="usa-button" type="submit">    
+  <img 
+    src="{% raw %}{{ uswds image path }}{% endraw %}/usa-icons-bg/search--white.svg" 
+    class="usa-search__submit-icon" 
+    alt="Search" />
+</button>
+```
+
+##### <span class="usa-tag bg-accent-cool-darker">Markup</span> Update social media icons in the footer.
+
+You'll need to update social media icons in the USWDS footer. We're now using explicit images to better support legibility in instances where icons do not load. 
+
+###### What to do
+
+1. Check your codebase for instances of `usa-social-link.`
+1. If you use it, Update the markup from the old version to the new version.
+
+###### Old code
+
+{:.site-terminal}
+```html
+<a class="usa-social-link usa-social-link--facebook" 
+  href="{% raw %}{{ link }}{% endraw %}">
+  <span>Facebook</span>
+</a>
+<a class="usa-social-link usa-social-link--twitter" 
+  href="{% raw %}{{ link }}{% endraw %}">
+  <span>Twitter</span>
+</a>
+<a class="usa-social-link usa-social-link--youtube" 
+  href="{% raw %}{{ link }}{% endraw %}">
+  <span>YouTube</span>
+</a>
+<a class="usa-social-link usa-social-link--instagram" 
+  href="{% raw %}{{ link }}{% endraw %}">
+  <span>Instagram</span>
+</a>
+<a class="usa-social-link usa-social-link--rss" 
+  href="{% raw %}{{ link }}{% endraw %}">
+  <span>RSS</span>
+</a>
+```
+
+###### New code
+
+{:.site-terminal}
+```html
+<a class="usa-social-link" 
+  href="{% raw %}{{ link }}{% endraw %}">
+  <img 
+    class="usa-social-link__icon" 
+    src="/usa-icons-bg/search--whi/usa-icons/facebook.svg" 
+    alt="Facebook" />
+</a>
+<a class="usa-social-link" 
+  href="{% raw %}{{ link }}{% endraw %}">
+  <img 
+    class="usa-social-link__icon" 
+    src="/usa-icons-bg/search--whi/usa-icons/twitter.svg" 
+    alt="Twitter" />
+</a>
+<a class="usa-social-link" 
+  href="{% raw %}{{ link }}{% endraw %}">
+  <img 
+    class="usa-social-link__icon" 
+    src="/usa-icons-bg/search--whi/usa-icons/youtube.svg" 
+    alt="YouTube" />
+</a>
+<a class="usa-social-link" 
+  href="{% raw %}{{ link }}{% endraw %}">
+  <img 
+    class="usa-social-link__icon" 
+    src="/usa-icons-bg/search--whi/usa-icons/instagram.svg" 
+    alt="Instagram" />
+</a>
+<a class="usa-social-link" 
+  href="{% raw %}{{ link }}{% endraw %}">
+  <img 
+    class="usa-social-link__icon" 
+    src="/usa-icons-bg/search--whi/usa-icons/rss_feed.svg" 
+    alt="RSS" />
+</a>
+```
```

Comment:
@thisisdano @bonnieAcameron Wondering if it would be helpful to link to the full release notes for each of these sections so that users can get more context if desired. 

Ex: https://github.com/uswds/uswds/releases/tag/v2.13.0

Reply:
agreed!

Reply:
@bonnieAcameron how do you want the copy to look for that?

Reply:
@mahoneycm my preference is that we keep it simple and just hyperlink the text to any version referenced. If I'm not quite answering your question can we grab some time Monday AM?

Reply:
@bonnieAcameron In this section the only mention of version numbers are on the accordion buttons that expand each section so we wouldn't be able to do that here. Maybe add _"Version 2.x.x release notes"_ to the top/bottom of each section?

Reply:
Oh, I see @mahoneycm! I misunderstood. I think in that case, let's not link out. It really should only be at the <p> level.